pele_platform/Adaptive/simulation.py

```diff
@@ -186,6 +187,16 @@ def run_adaptive(args: pv.EnviroBuilder) -> pv.EnviroBuilder:
         else:
             env.native = ""
 
+        #interaction restrictions
+        interaction_restrictions = ir.InteractionRestrictionsBuilder()
```

Comment:
This should be initialized inside the first branch of the `if`, as it's not used otherwise.